src/main/java/com/greghaskins/spectrum/Suite.java

```diff
@@ -20,22 +20,38 @@
   private final List<Child> children = new ArrayList<>();
   private final Set<Child> focusedChildren = new HashSet<>();
 
+  private final ChildRunner childRunner;
+
   private final Description description;
   private final Parent parent;
   private boolean ignored;
 
+  @FunctionalInterface
+  interface ChildRunner {
+    void runChildren(final Suite suite, final RunNotifier notifier);
+  }
```

Comment:
This allows the running of children to be a strategy which can vary. A sort of micro subclassing.